torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7875,6 +7887,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    decorators=(toleranceOverride({torch.float32: tol(atol=0, rtol=4e-6), }),),
                    dtypes=all_types_and(torch.bool),
                    safe_casts_outputs=True),
+    OpInfo(
+        "nn.functional.dropout",
+        ref=_NOTHING,
```

Comment:
Due to the random behavior of dropout, I don't think it is possible to have a black-box reference in `numpy`.